assignment.js

```diff
@@ -113,6 +150,27 @@ console.log('Problem 3:')
 
 // Add your code below this line
 
+let playerAChoice = "rock"
+let playerBChoice = "paper"
+
+if (playerAChoice === playerBChoice) {
```

Comment:
🛠 Read the comment below and update your code accordingly

@nurisellie  Looks like your structure in on the right path but the syntax for your test conditions is incorrect

Here you are trying to test if playerA has selected 'rock' and playerB has selected:
`playerAChoice === "rock", playerBChoice ==="paper"` 

^
this means something different in JavaScript [see the Comma operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comma_Operator) it's a rarely used operator and does not achieve what we're trying to do 

you need to use a logical AND to make this work: 

`playerAChoice === "rock" && playerBChoice ==="paper"`

[Check out the class slide on logical operators for more info](https://jsd20190730.github.io/02-javascript-basics/logical-operators)

Please update your code accordingly